### ISSUE TITLE
On Desktop take focus on task reminder only if option is selected

### ISSUE THREAD

**Issue by MathRPG** [super-productivity/super-productivity#1968]
Create an option in settings to specify app behavior when a task reminder is due: desktop notification, take focus back to main window, simply play a sound, or a combination of the three.

**Comment by guicattani** [comment]
Can this be reopened?
Sometimes I'm in mid writing a sentence and Super Prod gets the focus with the `Due Task` modal

**Comment by vctls** [comment]
Yes please. The focus steal is really obnoxious. I don't understand how anyone can like being interrupted in their work by a window suddenly stealing the focus. It's even worse when using multiple workspaces.
The app already sends desktop notifications, the focus steal is totally unnecessary and should be disabled by default.

Edit: even more obnoxious is the fact that if you dismiss the modal by clicking outside of it or pressing Esc, it pops back up three seconds later, stealing the focus again.

**Comment by johannesjo** [comment]
@vctls well, people are different. There quite a few requests asking for even more "obnoxious" behavior xD I guess it is matter of taste. So, yeah probably making this behavior configurable is the best way to go.
